healthcare-management/src/app/app.routes.ts

```diff
@@ -31,6 +31,12 @@ export const routes: Routes = [
       }
     ],
   },
+  {
+    path: 'symptom-checker',
+    loadComponent: () => import('./components/symptom-checker/symptom-checker.component').then(m => m.SymptomCheckerComponent),
+    canActivate: [AuthenticationGuard],
```

Comment:
AuthenticationGuard este folosit pentru cei care nu sunt conectati in cont. RoleGuard este folosit pentru a vedea ce user poate accesa pagina. Aici ca sa poti restrictiona doar ca pacientii sau doctorii sa acceseze aceasta pagina ai 2 solutii:

1 .fie adaugi si RoleGuard in canActivate (canActivate: [AuthenticationGuard, RoleGuard]) 
2. fie scoti data: {expectedRole: ['PATIENT', "DOCTOR"]} si presupunem ca orice user poate accesa aceasta pagina

Deocamdata nu o sa fie probleme daca e lasat asa, dar pe viitor s-ar putea sa apara bug-uri (ex: un rol care nu ar trebui sa aiba acces o sa aiba acces)